src/micromamba/install.cpp

```diff
@@ -183,7 +171,11 @@ install_specs(const std::vector<std::string>& specs, bool create_env, int solver
         sdir->load();
         multi_dl.add(sdir->target());
         subdirs.push_back(sdir);
-        if (ctx.strict_channel_priority)
+        if (ctx.channel_priority == ChannelPriority::kDisabled)
```

Comment:
i think it would be better to consider flexible and disabled the same way, and strict differently

Reply:
which is hte behavior from before

Reply:
`flexible` looks like a mix between `strict` for specs and `disabled` for deps.. I don't know